libs/frontend/src/lib/features/puzzles/components/play-puzzle.svelte

```diff
@@ -231,19 +232,11 @@
 						>
 							<div class="flex flex-col gap-4 lg:flex-row lg:gap-8">
 								<LogicalUnit class="w-full space-y-2 lg:max-w-[50%]">
-									<h3 class="text-lg font-semibold">Input</h3>
-
-									<div class="max-h-[20vh] w-full overflow-scroll rounded-lg border p-4">
-										<pre><code>{validator.input.trimEnd()}</code></pre>
-									</div>
+									<OutputBox title="Input">{validator.input.trimEnd()}</OutputBox>
 								</LogicalUnit>
 
 								<LogicalUnit class="w-full space-y-2 lg:max-w-[50%]">
-									<h3 class="text-lg font-semibold">Expected output</h3>
-
-									<div class="max-h-[20vh] w-full overflow-scroll rounded-lg border p-4">
-										<pre><code>{validator.output.trimEnd()}</code></pre>
-									</div>
+									<OutputBox title="Expected output">{validator.output.trimEnd()}</OutputBox>
```

Comment:
you're missing an h3 here somewhere, don't think that there is an h3 before this which would make it non valid html

Reply:
Oops, good catch! Maybe it's a sign that the heading shouldn't live inside the `OutputBox` component at all since it makes the component sort of dependent on context. What do you think?

Reply:
Uhmn, an h4 is pretty deep in the sauce, maybe it should just be a styled <p>? Not sure yet, gonna think about it today :pray:

Reply:
think a <p> is good enough for now for the thing in the outputbox, then we don't really need the h3 prior? :thinking: